bears/python/YapfBear.py

```diff
@@ -0,0 +1,137 @@
+import sys
+
+from coalib.bears.LocalBear import LocalBear
+from yapf.yapflib.yapf_api import FormatFile
+from coalib.results.Result import Result
+from coalib.bearlib.spacing.SpacingHelper import SpacingHelper
+from coalib.results.Diff import Diff
+from coalib.misc.ContextManagers import prepare_file
+from coalib.bears.requirements.PipRequirement import PipRequirement
+
+
+class YapfBear(LocalBear):
+    """
+    Check and correct formatting of Python Code using ``yapf`` utility.
+
+    See <https://github.com/google/yapf> for more information.
+    """
+    LANGUAGES = {"Python"}
```

Comment:
should also have more specific versions, like autopep8 bear has LANGUAGES = {"Python", "Python 2", "Python 3"}